pkg/detector/library/python/compare_test.go

```diff
@@ -0,0 +1,118 @@
+package python_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	dbTypes "github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy/pkg/detector/library/python"
+	"github.com/aquasecurity/trivy/pkg/log"
+)
+
+func TestPep440Comparer_MatchVersion(t *testing.T) {
```

Comment:
If I read it correctly matchVersion is a private method whereas IsVulnerable is exercised here.

```suggestion
func TestPep440Comparer_IsVulnerable(t *testing.T) {
```

Reply:
Nice catch! You're right. I've fixed it. Thanks.
https://github.com/aquasecurity/trivy/pull/816/commits/227062ab23d992afbcaf52c43cfef191482d15ef